universal-login-relayer/lib/relayer.ts

```diff
@@ -65,7 +65,7 @@ class Relayer {
     }));
     this.ensService = new ENSService(this.config.chainSpec.ensAddress, this.config.ensRegistrars, this.provider);
     this.authorisationService = new AuthorisationService(this.database);
-    this.walletContractService = new WalletService(this.wallet, this.config.walletMasterAddress, this.ensService, this.hooks);
+    this.walletContractService = new WalletService(this.wallet, this.config.walletMasterAddress, this.ensService, this.hooks, this.config.factoryAddress);
```

Comment:
I would consider passing `this.config` instead of both `this.config.walletMasterAddress` and `this.config.factoryAddress`.